src/main/cpp/subsystems/Climber.cpp

```diff
@@ -11,10 +11,19 @@ using namespace frc3512;
 using namespace frc3512::Constants::Climber;
 using namespace frc3512::Constants::Robot;
 
+Climber::Climber() {
+    // Create new NetworkTable
+    auto inst = nt::NetworkTableInstance::GetDefault();
+    auto table = inst.GetTable("Diagnostics");
+
+    m_climberEncoderEntry = table->GetEntry("Climber-Encoder");
+}
+
 void Climber::SetTransverser(double speed) { m_transverser.Set(speed); }
 
 void Climber::SetElevator(double speed) {
-    if (std::abs(speed) > 0.02) {
+    // Check if the position is less than the starting position
+    if (std::abs(speed) > 0.02 && m_limiter.GetPosition() >= 0.0) {
```

Comment:
This will cause the climber to never move again if the encoder position goes below zero. Try this instead:
```suggestion
    if (speed > 0.02 && m_limiter.GetPosition() <= 20.0 ||
        speed < 0.02 && m_limiter.GetPosition() >= 0.0) {
```